setup.py

```diff
@@ -39,14 +39,18 @@
         'sgp4',
         'requests',
     ],
-    tests_require=['logassert', 'flake8', 'hypothesis', 'mock', 'hypothesis[datetime]'],
```

Comment:
Do we need to add the requirements in both places? AFAIK `tests_require` is only used by setuptools when doing `python setup.py test`, and downloads the .eggs to a hardcoded location using `easy_install`. It's a bit painful (I'm thinking of removing it from all of my projects), so if you think we can remove them, I fully support doing so :)

Reply:
+1 thanks!

Reply:
I changed it, I need you to approve the MR to merge it

Reply:
For my review to unblock the merge I need write access to the repo!